demo/resources/views/demoWebpart.view.xml

```diff
@@ -0,0 +1,8 @@
+<view xmlns="http://labkey.org/data/xml/view" frame="none" title="To-Do List Webpart">
```

Comment:
This is a file generated by webpack, right?  I wonder if folks will take this as a sign that the `.view.xml` files should be checked in.

Reply:
Nope, this webpart one is not webpack generated. The entryPoints.js has a 'demoWebpart' entry which has `generateLib: true`. These related html/view.xml/webpart/xml files are added manually in this case for the webpart scenario.